src/Downloader.php

```diff
@@ -180,14 +180,20 @@ protected function fetchCertificates(string $hostName): array
             $connectTo = $hostName;
         }
 
-        $client = @stream_socket_client(
-            "ssl://{$connectTo}:{$this->port}",
-            $errorNumber,
-            $errorDescription,
-            $this->timeout,
-            STREAM_CLIENT_CONNECT,
-            $streamContext
-        );
+        try {
+            $client = stream_socket_client(
+                "ssl://{$connectTo}:{$this->port}",
+                $errorNumber,
+                $errorDescription,
+                $this->timeout,
+                STREAM_CLIENT_CONNECT,
+                $streamContext
+            );
+        }
+        catch (\ErrorException $ex) {
```

Comment:
Could you import this, so we can lose the `\`. Also rename `$ex` to `$exception`.